src/main/java/github/PullRequestServiceEx.java

```diff
@@ -0,0 +1,77 @@
+package github;
+
+import com.google.gson.reflect.TypeToken;
+import org.apache.logging.log4j.Logger;
+import org.eclipse.egit.github.core.IRepositoryIdProvider;
+import org.eclipse.egit.github.core.client.PagedRequest;
+import org.eclipse.egit.github.core.service.PullRequestService;
+import util.HTLog;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.eclipse.egit.github.core.client.IGitHubConstants.*;
+
+public class PullRequestServiceEx extends PullRequestService {
+    private static final Logger logger = HTLog.get(PullRequestServiceEx.class);
+
+    /**
+     * Gets a pull request's review comments
+     *
+     * @param repository
+     * @param pullRequestNumber
+     * @return list of review comments
+     * @throws IOException
+     */
+    public List<ReviewComment> getReviewComments(IRepositoryIdProvider repository,
+                                                 int pullRequestNumber) throws IOException {
+        return getReviewComments(repository, Integer.toString(pullRequestNumber));
+    }
+
+    /**
+     * Gets a pull request's review comments
+     *
+     * @param repository
+     * @param pullRequestNumber
+     * @return list of review comments
+     * @throws IOException
+     */
+    public List<ReviewComment> getReviewComments(IRepositoryIdProvider repository,
```

Comment:
Is this method needed? It doesn't ensure the PR number is an int. Maybe we should have the int constructor as the only interface for this functionality. This will allow us to skip some checks below too (only need one for `> 0`).